### ISSUE TITLE
Plan UI overhaul

### ISSUE THREAD

**Issue by breznak** [nupic-community/nupic.visualizations#72]
A lot of information is shown through the `handleError()` routine. The display may get cluttered with all the bubbles. Some way to make this more compact/clear? 

Ref.: https://github.com/nupic-community/nupic.visualizations/pull/69#issuecomment-171086137

**Comment by breznak** [comment]
I think first useful step (for devs) would be adding `console.log` to each `handleError` call? 

Do you think the amount of info shown through the notices is too much? ..I don't think it's that bad. Or we may just try to reduce the warnings we show (?)

**Comment by jefffohl** [comment]
Yes. I think I need to do an overhaul of the entire UI. There are a number of issues that have arisen lately as we have added more functionality.

**Comment by breznak** [comment]
for the UI overhaul, I'd like to suggest "just throwing all UI controls in" for now, and we can get feedback what is used, along with what,... and from there make a suitable UX?

**Comment by jefffohl** [comment]
How do you mean, "just throwing all UI controls in"? Do you mean, create UI controls for all conceivable settings?

I am noticing that there are a couple of stages in the process of parsing and visualizing a CSV file:
- There are options for before the user feeds the file into the system. For example, indicating whether the file will be continuously updated or not.
- There is feedback after the file has started loading - for example, whether or not forced windowing is being applied, due to the size of the file.
- After the file is loaded, or perhaps while the file is being streamed continuously, there are options - such as what series to show, threshold highlights, normalization, etc.

It would be good to take into account these different stages in the UX.

**Comment by breznak** [comment]
Oh, you have a better perspective than me, I was thinking just about the stage 3. And for there applies the "throw all in" for now.

**Comment by breznak** [comment]
Here all all UI/UX issues we have
https://github.com/nupic-community/nupic.visualizations/labels/UI%2FUX

(btw, will change the title to something more descriptive)

**Comment by breznak** [comment]
Idea for setup step: https://github.com/nupic-community/nupic.visualizations/issues/77#issuecomment-171709224 

I'd like to see similar show/hide btn "Mode", where users can switch to/toggle the following choices: 
- anomaly: #68 #31 
- Editor: #23 #24 #21 
- Monitoring
